core/src/main/java/org/apache/carbondata/core/datamap/TableDataMap.java

```diff
@@ -202,8 +202,9 @@ public CarbonTable getTable() {
                 null, table.getMinMaxCacheColumns(segmentProperties),
                 false);
         for (DataMap dataMap : dataMaps.get(segment)) {
-          pruneBlocklets.addAll(
-              dataMap.prune(filter.getResolver(), segmentProperties, partitions, filterExecuter));
+          pruneBlocklets.addAll(dataMap
```

Comment:
make format properly, move datamap to next line

Reply:
done